usb_port: '/dev/ttyUSB0'
boudrate: 9600
enable_timeout: false